shuup/simple_supplier/utils.py

```diff
@@ -98,7 +102,7 @@ def get_stock_adjustment_div(request, supplier, product):
     :rtype: str
     """
     latest_adjustment = StockAdjustment.objects.filter(
-        product=product, supplier=supplier).order_by("-created_on").first()
+        product=product, supplier=supplier, type=StockAdjustmentType.INVENTORY).order_by("-created_on").first()
```

Comment:
again, latest adjustment is usually last row added into database, hence the `.last()` should be enough here. But if there is a way to create `StockAdjustment` with custom `created_on`, ignore this :)

also you could have `ordering = "-created_on"` in object `Meta` imo and then use the `.first()` here.